src/commands.ts

```diff
@@ -10,9 +10,10 @@ import { ResourceModels, ResourceManagementClient, SubscriptionClient, Subscript
 import DocumentdbManagementClient = require("azure-arm-documentdb");
 import docDBModels = require("azure-arm-documentdb/lib/models");
 import { DocumentClient } from 'documentdb';
+import { parse } from 'jsonc-parser';
```

Comment:
Currently, the only use case of this is to ignore comments in the JSON. JSON.parse does not work well with comments.

Reply:
Agreed. 
Should we be adding this behavior in case the user adds comments themselves? 
Currently, we would be throwing an error of the form : 
 'Unexpected token / in JSON at position ...'

Reply:
Personally, I don't think the user will add comments to the file and I don't think the added complexity of using a third party tool (and testing to make sure it works) is worth it.

But I see your point so it's up to you